dragonchain/webserver/routes/interchain.py

```diff
@@ -96,6 +103,19 @@ def create_ethereum_interchain_v1() -> Tuple[str, int, Dict[str, str]]:
     return helpers.flask_http_response(201, interchain.create_ethereum_interchain_v1(data))
 
 
+@request_authorizer.Authenticated()
+def create_binance_interchain_v1() -> Tuple[str, int, Dict[str, str]]:
+    if not flask.request.is_json:
+        raise exceptions.BadRequest("Could not parse JSON")
+    data = flask.request.json
+    try:
+        _validate_binance_network_create_v1(data)
+    except fastjsonschema.JsonSchemaException:
+        raise exceptions.ValidationException("User input did not match JSON schema")
```

Comment:
don't use generic error. Pass back error string like the rest of the api

Reply:
(Refer to other examples in rebased code to fix.)

Reply:
Fixed.